<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="mainWindow">
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="title">
      <string>Event Registration Form</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="layoutHorizontalMain">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="layoutVerticalInfo">
          <item>
           <layout class="QHBoxLayout" name="layoutH1">
            <item>
             <widget class="QLabel" name="labelFieldRequired">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <italic>false</italic>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Required Fields</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutH2">
            <item>
             <widget class="QLabel" name="labelFName">
              <property name="text">
               <string>First Name: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textFName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutH3">
            <item>
             <widget class="QLabel" name="labelLName">
              <property name="text">
               <string>Last Name: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textLName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutH4">
            <item>
             <widget class="QLabel" name="labelEmail">
              <property name="text">
               <string>Email Address: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textEmail"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutH5">
            <item>
             <widget class="QLabel" name="labelFieldOptional">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <italic>false</italic>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Optional Fields</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutH6">
            <item>
             <widget class="QLabel" name="labelPhone">
              <property name="text">
               <string>Phone Number: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textPhone"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutH7">
            <item>
             <widget class="QLabel" name="labelAge">
              <property name="text">
               <string>Age: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textAge"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="buttonClearForm">
              <property name="text">
               <string>Clear Form</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutVerticalInput">
          <item>
           <layout class="QVBoxLayout" name="layoutVerticalDisplay">
            <item>
             <widget class="QLabel" name="labelFieldDisplay">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <italic>false</italic>
                <bold>true</bold>
                <underline>false</underline>
               </font>
              </property>
              <property name="text">
               <string>Display</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="displayText">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layoutDisplayButtons">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="buttonShowList">
                <property name="text">
                 <string>Current List</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonClearDisplay">
                <property name="text">
                 <string>Clear Display</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layoutVerticalButtons">
            <item>
             <widget class="QLabel" name="labelFieldButtons">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Confirm</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layoutHButtons">
              <item>
               <widget class="QPushButton" name="buttonSubmit">
                <property name="text">
                 <string>Submit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonSave">
                <property name="text">
                 <string>Save All Data</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
